packages/web-components/src/menu-item/menu-item.styles.ts

```diff
@@ -41,46 +42,49 @@ export const styles = css`
     cursor: pointer;
     overflow: visible;
     contain: layout;
+    --indent: 0;
   }
 
   :host(:hover) {
     background: ${colorNeutralBackground1Hover};
   }
 
+  :host(:focus-visible) {
+    border-radius: ${borderRadiusMedium};
+    outline: 2px solid ${colorStrokeFocus2};
+  }
+
+  .indicator {
+    width: 20px;
+  }
+
   .content {
     white-space: nowrap;
     flex-grow: 1;
     grid-column: auto / span 2;
     padding: 0 2px;
   }
 
-  .checkbox,
-  .radio {
-    display: none;
-  }
-
-  .input-container,
-  .expand-collapse-glyph-container,
+  .indicator,
+  .submenu-glyph,
   ::slotted([slot='start']),
-  ::slotted([slot='end']),
-  :host([checked]) .checkbox,
-  :host([checked]) .radio {
+  ::slotted([slot='end']) {
     display: inline-flex;
     justify-content: center;
     align-items: center;
```

Comment:
I think we can drop these if we flatten the DOM for the indicator/submenu-glyph.